pyproject.toml

```
@@ -40,6 +40,7 @@ dependencies = [
   "build",
   "click >= 8",
   "pip >= 22.2",
+  "toml >= 0.10.1; python_version<'3.11'",
```

Let's make it a little bit more sparse, like the first part.
```suggestion
  "toml >= 0.10.1; python_version < '3.11'",
```

---

with the recent comments in the Python modules, this should be
```suggestion
  "tomli >= 0.10.1; python_version < '3.11'",
```